src/components/VKButton.tsx

```diff
@@ -0,0 +1,19 @@
+import getVKCode from "../helpers/parseUrl";
+import { REDIRECT_URL, VK_APP, BACKEND_URL } from "../constants";
+
+const VKButton = () => {
+  const handleRedirect = () => {
+    window.location.href = `https://oauth.vk.com/authorize?client_id=${VK_APP.id}&display=popup&redirect_uri=${REDIRECT_URL}&scope=email&response_type=code&v=5.120&state=4194308`;
+    const code = getVKCode(window.location.href)!;
+    if (code) console.log(code);
```

Comment:
Please write if with blocks:

if (code) {
...
} else if {
...
}


It is highly recommended that you always use braces because if you (or someone else) ever expands the statement it will be required.

Reply:
done

Reply:
You can not write 'done'. Just push 'Resolve conversation' button instead